app/controllers/api/v1/accounts/conversations_controller.rb

```diff
@@ -80,6 +80,12 @@ def toggle_typing_status
     head :ok
   end
 
+  def update_draft_message
+    @conversation.draft_message_content = params.permit(:message)[:message]
```

Comment:
Saving it as an attribute to conversation is not ideal. There are multiple people looking at the same conversation. Showing a draft saved by one of the users to another user is not correct.